snakemake/03_generate_contigs.smk

```diff
@@ -1,6 +1,6 @@
 configfile: 'variant_calling.yaml'
 output_folder='/opt/analysis'
-log_folder=config['output_directory']+'/run_settings'
+log_folder='/opt/analysis/run_settings'
```

Comment:
makes sense - since snakemake is now inside the sif, run settings are now getting sent to a sif file internal location